### DISCUSSION TITLE
File isn't being copied over in dotfiles

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#29574] [Codespaces] [answered]
Hey! I use a prompt called starship, and there is a config file called `starship.toml`. In my `install.sh`, I am copying over the local `starship.toml` to `$HOME/.config/starship.toml`, which is the default path for starship configuration. When cding into the `.config` directory, I have found that the file is not there. Relevant file: https://github.com/Milo123459/dotfiles/blob/master/install.sh

**Comment by thomas-sickert** [comment, marked as answer]
Hey @Milo123459! Sorry you're running into issues. Could you try following the steps in [this troubleshooting guide](https://docs.github.com/en/codespaces/troubleshooting/troubleshooting-dotfiles-for-codespaces) to get some more details on whether the dotfile installation is running correctly?

**Reply by Milo123459** [reply]
I know it's running correctly as my `.bashrc` is being copied over. I have also checked and they are being cloned. I searched logs for any `cp` errors but couldn't see any.

**Reply by thomas-sickert** [reply]
I just forked the repo and set it to my dotfiles repo. This is the error I get when I run `cat /workspaces/.codespaces/.persistedshare/creation.log | grep cp`:

```
2022-08-15 21:13:29.458Z: cp: cannot create regular file '/home/codespace/.config/starship.toml': No such file or directory
```

**Reply by Milo123459** [reply]
Does .config not exist? I'm confused.

**Reply by Milo123459** [reply]
Making it as if it didn't exist worked! Thank you for the help!
